CONTRIBUTING.md

```diff
@@ -144,6 +144,20 @@ When submitting a PR, please keep the following guidelines in mind:
 2. Please link the issue you worked on in the description of your pull request and in your commit message. For example, for issue-125, the PR description and commit message could be: ```Go through TODOs in the code and create issues for them  
 Fixes #125```
 
+3. Your PR should include [docstrings](https://google.github.io/styleguide/pyguide.html) for all functions and classes, as well as comments to help make your code easier to understand.
```

Comment:
Do we want to mention the link to the documentation?

Reply:
The link to the docs are mentioned in step 5.